src/App.tsx

```diff
@@ -24,51 +72,28 @@ export const App: React.FC = () => {
 
           <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
             <div className="dropdown-content">
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Bernard Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter Antone Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Haverbeke</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-link">Pieter de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Hercke</p>
-              </div>
+              {filteredPeople.length === 0 ? (
+                <div className="dropdown-item" data-cy="no-suggestions-message">
+                  <p className="has-text-danger">No matching suggestions</p>
+                </div>
+              ) : (
+                filteredPeople.map(person => (
+                  <div
+                    key={`${person.slug}-${person.born}`}
```

Comment:
The key generation now uses `person.slug` and `person.born`, which is better than using `slug` alone. However, ensure that this combination is unique for each person. If `slug` and `born` together are not guaranteed to be unique, consider using a more reliable unique identifier.